### PR TITLE
WIP to improve DevTools Lighthouse scores

### PR DESCRIPTION
**Pull request by silviubogan**
Maybe useful one day: https://gist.github.com/silviubogan/5d01886907d3b0ec981b25dc06d1eeda.

### CODE REVIEW

File: src/helpers/Html/Html.jsx [hunk 1]
```diff
@@ -82,7 +84,7 @@ class Html extends Component {
           <meta name="viewport" content="width=device-width, initial-scale=1" />
           <meta name="apple-mobile-web-app-capable" content="yes" />
           {/* Add the crossorigin while in development */}
-          {extractor.getLinkElements().map((elem) =>
+          {extractor.getLinkElements({ rel: 'preload' }).map((elem) =>
```
Comment (tiberiuichim) [inline src/helpers/Html/Html.jsx:87]: I don't think this is needed, make sure to test in "production" mode by running "yarn build; yarn start:prod"

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

**Comment by tisto** [comment]
@silviubogan @tiberiuichim nice! At kitconcept we integrate lighthouse (and some other perf tests) to continuously monitor our performance over time. We could do the same for Volto core (this has been on my todo list for quite some time actually). The only problem we have to overcome is to automatically deploy the latest version of Volto (master) somewhere as part of our CI job so that a performance test can run against this instance.

File: src/helpers/Html/Html.jsx (hunk 1 above)
Comment (silviubogan) [reply src/helpers/Html/Html.jsx:87]: I think it helps: https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content. I tested with the given commands. The score goes from 17 to 16 on my laptop when compared to Volto master, but it is WIP.

**Comment by cekk** [comment]
@tisto have you automated it in your projects or do you run perf tests manually when needed?

Btw we started looking into Volto perf problems too and this pr could be a good starting point ;)

**Comment by tiberiuichim** [comment]
See also https://github.com/plone/volto/issues/2025

**Comment by tisto** [comment]
@cekk we auto deploy to a dev server after all tests have been passed. After the deployment we run a set of perf. tests against the dev instance (that contains example content that always stays the same). We then include the lighthouse HTML reports in Jenkins. I am happy to help with setting this up. Though, we have to solve the problem of the auto-deployment first.

**Comment by tisto** [comment]
@sneridagh what if we set up a small hetzner VM where we auto-deploy the latest volto master automatically on every commit? With this we could at least track the performance over time on master (not on PRs and branches though).

I am open to any other thoughts. Ideally we would auto-deploy every single PR and run lighthouse against it and add the lighthouse score in the GH pull request. I could take care of the CI part but I am no expert on the auto-deployment part.

cc @cekk

**Comment by mtoepfl** [comment]
Perhaps Vercel can be used. Should not matter that Volto doesn't depend on NextJS. RestAPI can be hosted separate (CORS or proxy), or not? They offer a plan for Open Source projects:

https://vercel.com/pricing

**Comment by tiberiuichim** [comment]
Closed, there's critical css support now in Volto 10.10

**Pull request closed without merging**